BLDC.py

```diff
@@ -0,0 +1,39 @@
+import GPIO
+import PWM
+
+pwmCtrl = PWM.signal()
```

Comment:
why does this need to be global? looks like you can just initialize this in the `Motor` constructor.